### PR TITLE
Update publishing-extension.md

### PR DESCRIPTION
**Pull request by seaniyer**
Adding pre-requisites for domain verification

### CODE REVIEW

**Comment by seaniyer** [comment]
FYI @isidorn @mariaghiondea @skofman1

File: api/working-with-extensions/publishing-extension.md [hunk 1]
```diff
@@ -305,7 +305,7 @@ See more information in [Marketplace Presentation Tips](/api/references/extensio
 
 ### Verify a publisher
 
-You can become a **verified publisher** by verifying ownership of an [eligible domain](#eligible-domains) associated with your brand or identity. Once your publisher is verified, the Marketplace will add a verified badge to your extension details.
+You can become a **verified publisher** by verifying ownership of an [eligible domain](#eligible-domains) associated with your brand or identity. Once your publisher is verified, the Marketplace will add a verified badge to your extension details. As pre-requisites to get approval, the publisher must have an extension on the VS Marketplace for at least 6 months, and the domain submitted for verification must also be at least 6 months old. Please wait until these critera are met before applying for verification.
```
Comment (mariaghiondea) [inline api/working-with-extensions/publishing-extension.md]: I believe that we should keep the items required in the same place - either here or line 339 to make it easier for folks to find. We could also add a link here to the items needed.

Comment (seaniyer) [reply api/working-with-extensions/publishing-extension.md]: Yes, makes sense to keep in one place. My thinking is to provide the prerequisite info upfront before users take any steps to apply. So, I'll remove it from line 339 unless there are any reasons to do differently.

**Review by mariaghiondea — APPROVED** [review]

File: api/working-with-extensions/publishing-extension.md (hunk 1 above)
Comment (mariaghiondea) [reply api/working-with-extensions/publishing-extension.md]: looks good. thank you for the changes.

**Review by ntrogh — APPROVED** [review]
LGTM. @seaniyer Let me know when you want this merged.

**Review by chrmarti — APPROVED** [review]

**Comment by isidorn** [comment]
@ntrogh let's go ahead and merge this in.
Thank you Sean!

**Pull request merged**